lib/modules/manager/crossplane/readme.md

```diff
@@ -0,0 +1,40 @@
+The `crossplane` manager has no `fileMatch` default patterns, so it won't match any files until you configure it with a pattern.
```

Comment:
```suggestion
To use the `crossplane` manager you must set your own `fileMatch` pattern.
This is because the `crossplane` manager has no default patterns for the `fileMatch`.
The `crossplane` manager only matches files when you set a `fileMatch` pattern.
```

We should use the word _must_ to indicate requirements. 😉